charts/pega/templates/_pega-deployment.tpl

```diff
@@ -86,6 +86,10 @@ spec:
         - name: REQUESTOR_PASSIVATION_TIMEOUT
           value: "{{ .node.requestor.passivationTimeSec }}"
 {{- end }}
+{{ if and .root.Values.constellation (eq .root.Values.constellation.enabled true) }}
+        - name: NODE_SETTINGS
```

Comment:
Should NODE_SETTINGS only apply if constellation is enabled?  We may need to find a different way of passing this via a dedicated parameter.

Reply:
Yes @dcasavant we need it only for constellation. And we can remove "if" condition, as there is no harm in setting that DSS(DSS name= CONSTELLATIONSVCURL). 
Please let us know your thoughts on this.

DSS set value via NODE_SETTINGS : PEGA-UIENGINE/CONSTELLATIONSVCURL=/prweb/constellation"

Reply:
Needs a dedicated (not `NODE_SETTINGS`) param.

Reply:
Can be added to https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-conf/Catalina/localhost/prweb.xml